services/blockchain-coordinator/shared/scheduler.js

```diff
@@ -263,9 +265,9 @@ const scheduleMissingBlocksIndexing = async () => {
 
 				if (result.length === 0) {
 					const lastIndexVerifiedHeight = await getIndexVerifiedHeight();
-					if (batchEndHeight === lastIndexVerifiedHeight + MAX_QUERY_RANGE) {
```

Comment:
`MAX_QUERY_RANGE` > `MAX_BLOCKS_TO_SCHEDULE`, so we will always have only 1 batch. The values needs adjustment.